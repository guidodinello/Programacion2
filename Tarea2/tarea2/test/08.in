# Implementar kesimo
kesimo 0
kesimo 1
insertarAlFinal (3,6.8)
kesimo 0
kesimo 1
infoCadena
kesimo 2
finalCadena
insertarAlFinal (5,3.4)
insertarAlFinal (8,9.4)
insertarAlFinal (9,2.6)
kesimo 1
infoCadena
kesimo 2
infoCadena
kesimo 4
infoCadena
kesimo 5
Fin
