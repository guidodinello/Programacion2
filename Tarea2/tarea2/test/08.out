1>#  Implementar kesimo.
2>loc quedó no válido.
3>loc quedó no válido.
4>Insertado al final.
5>loc quedó no válido.
6>loc en la posición 1.
7>(3,6.80)
8>loc quedó no válido.
9>loc al final.
10>Insertado al final.
11>Insertado al final.
12>Insertado al final.
13>loc en la posición 1.
14>(3,6.80)
15>loc en la posición 2.
16>(5,3.40)
17>loc en la posición 4.
18>(9,2.60)
19>loc quedó no válido.
20>Fin.
